lib/rom/lint/linter.rb

```diff
@@ -1,5 +1,19 @@
 module ROM
   module Lint
+    # Base class for building linters that check source code
+    #
+    # Linters are used by authors of ROM adapters to verify that their
+    # integration comnplies with the ROM api.
```

Comment:
Thanks for noting. This line was copy/pasted from the original commit message when this file was added.